extension/source/background/KeyringController.ts

```diff
@@ -110,6 +115,19 @@ export default class KeyringController {
       return networkData.address;
     },
 
+    createTempAccount: async (_request) => {
+      const pKey = generateRandomHex(256);
```

Comment:
Use a `BN254` spaced key instead of just a random hex. See https://github.com/thehubbleproject/hubble-bls/blob/master/src/mcl.ts#L186 . We should expose that via the signer in the client library somewhere/

Reply:
Ahh interesting we do `generateRandomHex` a couple of other places in the keyring controller which is where I copied this from.  Should I also switch those?

Also, just curious, what is the benefit of `BN254` over a random hex?

Reply:
Just want to double check that this is the right way to call that function

```
randFr().serializeToHexStr()
```

Reply:
yeah makes sense 

> what is the benefit of BN254 over a random hex?

If I understand correctly, The random number generated might sometimes lie outside the range of the BN curve we are using. This function ensures it's in range.

Reply:
@kautukkundan correct. If the key is out of range, we just hash it into a key that is in range. I have a bucklist item/change that will be more explicit about this in the future https://github.com/thehubbleproject/hubble-bls/issues/30